### DISCUSSION TITLE
is it possible to disable auto-capture?

### DISCUSSION THREAD

**Discussion by mross** [kangyu-california/PersistentWindows#255] [Q&A]
I would find it useful to have a companion setting to the "pause auto-restore" found in the task tray options menu, and that wold be the ability to disable auto-capture. 
By now I know when the displayport hot-plug problem is going to bite me, and that' s the only time i need to take a snapshot for manual restoration at a later time.

Anyone's thoughts on this much appreciated.

**Comment by kangyu-california** [comment]
So the request is to run PW in a semi-automatic way by capture manually and restore automatically?
I think the idea is doable.

**Comment by mross** [comment]
Thanks for getting back. To be clear, my goal is to be able to capture manually and restore manually -- but it is already possible to "pause" automatic restorations so nothing new is required on that side.

thanks!

**Comment by kangyu-california** [comment]
Once "pause auto restore" is active, PW immediately stop automatic capture.
i.e. "pause auto restore" implies "pause auto capture"

**Comment by mross** [comment]
ah, got it. Thanks for the explanation.
